src/webviews/src/settings/CronChooser.tsx

```diff
@@ -13,145 +20,144 @@ const defaultSchedules = {
   day: '0 0 * * *',
 }
 
+const CronFallback = ({ error }: { error: any }) => {
+  return (
+    <div className="flex items-center space-x-1 text-[color:var(--vscode-errorForeground)]">
+      <i className="codicon codicon-error text-sm pt-px" />
+      <p className="text-[12px]">{error}</p>
+    </div>
+  )
+}
+
+const ValidateCron = ({ value }: { value: string }) => {
+  const feedback = cronstrue.toString(value)
+  return (
+    <div className="flex items-center space-x-1 text-[#73C991]">
+      <i className="codicon codicon-pass pr-1 text-sm pt-px" />
+      <p className="text-[12px]">
+        Will run:{' '}
+        {feedback === 'Every minute' ? 'Every five minutes' : feedback}
+      </p>
+    </div>
+  )
+}
+
 const determineInitialCustomValue = (schedule?: string) => {
   if (!schedule) return false
   return !Object.values(defaultSchedules).includes(schedule)
 }
 
-const CronChooser: FunctionComponent<CronChooserProps> = props => {
-  const isInitiallyCustom = determineInitialCustomValue(props.value)
+const CustomCronTextField = ({
+  value,
+  onChange,
+}: {
+  value: string
+  onChange: (val: string) => void
+}) => {
+  const [localValue, setValue] = useState(value)
 
-  const [customCron, setCustomCron] = React.useState(
-    isInitiallyCustom ? props.value : defaultSchedules.day
+  return (
+    <div>
```

Comment:
<img width="433" alt="CleanShot 2022-01-31 at 16 36 54@2x" src="https://user-images.githubusercontent.com/25163139/151896175-861da802-0855-49b8-993a-fad7690956fc.png">

Why not use the helper text? The green text felt a bit out of place/alignment here.

Consider placing the link below the input + caption? If not, it should probably at least be aligned with the text input rather than the input + label as a group.